docker/ubuntu_20.04-dev.dockerfile

```
@@ -108,58 +110,51 @@ ENV LD_LIBRARY_PATH $HOME/opt/moab/lib:$LD_LIBRARY_PATH
 ENV LIBRARY_PATH $HOME/opt/moab/lib:$LIBRARY_PATH
 ENV PYTHONPATH=$HOME/opt/moab/lib/python${py_version}/site-packages/
 
+FROM moab AS dagmc
 # build/install DAGMC
-ARG build_dagmc="NO"
 ENV INSTALL_PATH=$HOME/opt/dagmc
-RUN if [ "$build_dagmc" = "YES" ]; then \
-        cd /root \
-        && git clone --depth 1 --branch stable https://github.com/svalinn/DAGMC.git \
-        && cd DAGMC \
-        && mkdir bld \
-        && cd bld \
-        && cmake .. -DMOAB_DIR=$HOME/opt/moab \
-                 -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH \
-                 -DBUILD_STATIC_LIBS=OFF \
-                 -DBUILD_UWUW=OFF \
-                 -DBUILD_TALLY=OFF \
-                 -DBUILD_MAKE_WATERTIGHT=OFF \
-                 -DBUILD_OVERLAP_CHECK=OFF \
-                 -DBUILD_TESTS=OFF \
-        && make -j 3\
-        && make install \
-        && cd ../.. \
-        && rm -rf DAGMC; \
-    fi
+RUN cd /root \
+    && git clone --depth 1 --branch stable https://github.com/svalinn/DAGMC.git \
+    && cd DAGMC \
+    && mkdir bld \
+    && cd bld \
+    && cmake .. -DMOAB_DIR=$HOME/opt/moab \
+                -DCMAKE_INSTALL_PREFIX=$INSTALL_PATH \
+                -DBUILD_STATIC_LIBS=OFF \
+                -DBUILD_UWUW=OFF \
+                -DBUILD_TALLY=OFF \
+                -DBUILD_MAKE_WATERTIGHT=OFF \
+                -DBUILD_OVERLAP_CHECK=OFF \
+                -DBUILD_TESTS=OFF \
+    && make -j 3\
+    && make install \
+    && cd ../.. \
+    && rm -rf DAGMC; \
```

Get rid of extra continuation info

```suggestion
    && rm -rf DAGMC
```